src/testFixtures/java/org/opensearch/neuralsearch/BaseNeuralSearchIT.java

```diff
@@ -691,6 +691,36 @@ protected void addSparseEncodingDoc(
         assertEquals(request.getEndpoint() + ": failed", RestStatus.CREATED, RestStatus.fromCode(response.getStatusLine().getStatusCode()));
     }
 
+    protected void addDocsThroughBulk(
```

Comment:
Do we really need the `id` field from the docs? If not, how about making `docs` a list of strings? 
From this function, we implicitly assert that every element in the list contains two keys, i.e., `id` and `text`. Other users might be confused with the type of `docs`

Reply:
We also want to verify if a certain doc exists after bulk action through querying doc by known id.

Reply:
See my comment below so that we do not need to query a document with `doc id`.